include/util_quda.h

```diff
@@ -140,20 +140,19 @@ namespace quda {
 
 #endif // MULTI_GPU
 
-
-#define checkCudaErrorNoSync() do {                    \
-  cudaError_t error = cudaGetLastError();              \
-  if (error != cudaSuccess)                            \
-    errorQuda("(CUDA) %s", cudaGetErrorString(error)); \
-} while (0)
-
+#define checkCudaErrorNoSync()                                                                                         \
+  do {                                                                                                                 \
+    qudaError_t error = cudaGetLastError();                                                                            \
+    if (error != qudaSuccess) errorQuda("(CUDA) %s", cudaGetErrorString(error));                                       \
+  } while (0)
 
 #ifdef HOST_DEBUG
 
-#define checkCudaError() do {  \
-  cudaDeviceSynchronize();     \
-  checkCudaErrorNoSync();      \
-} while (0)
+#define checkCudaError()                                                                                               \
+  do {                                                                                                                 \
+    qudaDeviceSynchronize();                                                                                           \
+    checkCudaErrorNoSync();                                                                                            \
+  } while (0)
```

Comment:
I wonder if it makes more send just to move these error checks into the target architecture files.  E.g., define the macro `checkQudaErrorNoSync` and `checkQudaError` and have these macros be defined directly in the target arch headers.  Just a thought.  Either way, you probably want to rename `Cuda` to `Quda`.

Reply:
Good idea, will implement.